src/ui/zcl_abapgit_gui_chunk_lib.clas.abap

```diff
@@ -812,6 +813,15 @@ CLASS zcl_abapgit_gui_chunk_lib IMPLEMENTATION.
                               && |{ lo_repo_online->get_url( ) }|
                       iv_class = |url| ).
 
+      IF iv_show_edit = abap_true.
+        ri_html->add_a( iv_txt   = ri_html->icon( iv_name  = 'edit-solid'
+                                                  iv_class = 'pad-sides'
+                                                  iv_hint  = 'Change remote' )
+                        iv_act   = |{ zif_abapgit_definitions=>c_action-repo_remote_settings }?| &&
+                                   |key={ io_repo->get_key( ) }|
+                        iv_class = |url| ).
+      ENDIF.
+
```

Comment:
Looks like the changes from #4723 are included in the first commit?

Reply:
Repo_view is in both. I didn't want a conflict. But this one doesn't  have the css and icon mime. I can combine the PRs is you like

Reply:
It'd be easier if you combined I think in this case (in a separate commit) easier to review as a whole then.
Will test this sometime today

Reply:
Done. If you test without #4721, then there will be an empty spot next to the URL but you should be able to click it. Or simply pull both #4721 and #4724.